runtime/src/main/java/org/corfudb/runtime/view/RuntimeLayout.java

```diff
@@ -49,7 +49,7 @@
      */
     public RuntimeLayout(@Nonnull Layout layout, @Nonnull CorfuRuntime corfuRuntime) {
 
-        this.layout = new Layout(layout);
+        this.layout = layout;
```

Comment:
I thought the whole point of a runtime layout is to have your own layout.

Reply:
It still does right. It's just the management agent which modifies layouts. We can make sure all modifications are made on a copy.

Reply:
Yeah, this is fine. a runtime layout is a runtime+layout so you can easily get new clients.